client/resources/original_messages.json

```diff
@@ -27,6 +27,39 @@
     "description": "The title of a page giving details about the app.",
     "message": "About"
   },
+  "add_access_key_dialog_header": {
+    "description": "The title of a dialog box that prompts the user to add a new access key.",
+    "message": "Add VPN access key"
```

Comment:
Let's go back to the original "Add access key".

"VPN" is confusing, and some consider it not a VPN (we don't give access to a private network). Also, we are moving towards a world where we may include proxyless. We are adding a strategy, not a server of service, hence keeping "access key".